docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -120,3 +130,95 @@ This will download test data and direct `data/current` symlink to the test data
 
 To switch back to using ScPCA data, rerun the script with the `--release current` option.
 If you already have the most recent data, this will not repeat downloading the data you already have.
+
+
+## Accessing ScPCA module results
+
+Once an analysis modules matures, the Data Lab ports the analysis to a separate repository called [`OpenScPCA-nf`](https://github.com/AlexsLemonade/OpenScPCA-nf).
+This repository holds a [Nextflow workflow](https://www.nextflow.io/) to reproducibility run and generate results from completed analysis modules.
+
+If you wish to use output from a completed analysis module in your module, you will need to obtain the results generated by this workflow.
+
+We have provided a [`download-results.py` script](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/download-results.py) that you can use to download all results from a module that has been run in `OpenScPCA-nf`.
+Results are available at specific tagged releases. <!-- STUB_LINK to openscpca-nf docs -->
+
+We briefly cover some of the most common use cases below, but we encourage you to review all the options available to you.
+
+You can list all the options for `download-results.py` by running the following from the root directory of the repository:
+
+```sh
+./download-results.py --help
+```
+
+!!! tip "No `--profile` necessary on Lightsail for Research"
+    Omit `--profile openscpca` from the commands below if you're using Lightsail for Research.
+
+Example script usage below assumes you have created a profile called `openscpca` when [configuring the AWS CLI](../../technical-setup/environment-setup/configure-aws-cli.md).
```

Comment:
We should note that before running these scripts they will also need to run `aws sso login --profile openscpca` (If we don't say that for the data download script, we should add it there too).

Reply:
For easier typing, I wonder if we should tell people to run the following commands to start:
```
export AWS_PROFILE=openscpca
aws sso login
```
and then leave off the `--profile` option in all of the examples.

Reply:
Just seeing these comments after pushing this (of course) https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/503/commits/d21aa10dc2058653d3c8117babb79ab252e924e0

I think it's good to keep the docs I added, but I agree with your suggestion here so will nix the admonitions in favor of a small section at the top "before running any script"